src/Property.ts

```diff
@@ -147,7 +147,16 @@ export default class Property<T = any> {
 		return this.getString().replace(/[0-9\.\-]/g, '');
 	}
 
-	getPixels(axis?: Axis, processPercent = false) {
+	getPixels(axis?: Axis, processPercent?: boolean): number;
+	getPixels(isFontSize?: boolean): number;
+	getPixels(axisOrIsFontSize?: Axis | boolean, processPercent = false): number {
```

Comment:
@dangreen linter complained had to add `: number` return type 🤷‍♂

Reply:
It's ok, my mistake